src/common/com/intellij/plugins/haxe/model/type/HaxeGenericResolver.java

```diff
@@ -76,6 +77,17 @@ public ResultHolder resolve(String name) {
     return results;
   }
 
+  @NotNull
+  public ResultHolder[] getSpecificsFor(@Nullable HaxeClassReference clazz) {
+    return getSpecificsFor(clazz != null ? clazz.getHaxeClass() : null);
```

Comment:
Whoops!  This is a bug: returning null.  I will change it to: `return getSpecificsFor(clazz != null ? clazz.getHaxeClass : ResultHolder.EMPTY);`